stories/feed-input/feed-input.stories.js

```diff
@@ -0,0 +1,90 @@
+import '../../dist/textarea.css';
+import '../../dist/feed-input.css';
+
+export default {
+    title: 'Components/Feed Input',
+    parameters: {
+        description: `
+        The Feed Input is a component specifically design to allow the user to type his/her own notes and add them to a Feed.
+`,
+        tags: ['f3', 'a11y', 'theme']
+    }
+};
+
+export const empty = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input empty">
+        <div class="fd-feed-input__thumb" style="background-image: url(http://lorempixel.com/460/620/people/7/)" role="img" title="Thumbnail"></div>
+        <textarea class="fd-textarea fd-feed-input__textarea" id="textarea-1" rows="1" placeholder="Post something here" aria-label="Feed"></textarea>
+        <button class="fd-button sap-icon--feeder-arrow fd-feed-input__submit-button" aria-label="Submit" title="Submit" aria-disabled="true" disabled></button>
+    </div>
+`;
+
+empty.storyName = 'Empty';
+empty.parameters = {
+    docs: {
+        storyDescription: 'Example with an empty text field'
+    }
+};
+
+
+export const notEmpty = () => `
+    <div class="fd-feed-input" role="region" aria-label="Feed input with one row text">
```

Comment:
and add this as a description "**The input with text displays some text within the field, which activates the submit button.**"